browser/plugins/audio_spatial_listener.plugin.js

```diff
@@ -0,0 +1,95 @@
+(function() {
+
+var SpatialListener = E2.plugins.audio_spatial_listener = function(core, node) 
+{
+	this.desc = 'Spatial Listener.';
+	
+	this.input_slots = [ 
+		{ name: 'position', dt: core.datatypes.VECTOR, desc: 'Position.', def: core.renderer.vector_origin },
+		{ name: 'forward', dt: core.datatypes.VECTOR, desc: 'Forward Vector.', def: core.renderer.vector_origin },
+		{ name: 'up', dt: core.datatypes.VECTOR, desc: 'Up Vector.', def: core.renderer.vector_origin }
```

Comment:
Default should be 0,1,0 probably - avoids you having to override this each time.
Also, isn't up always 0,1,0?